tests/test_wsbfev1.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
```

Comment:
declartion of utf-8 is not necesary, because the source code of files at python3 is utf-8 by default. https://docs.python.org/3/howto/unicode.html#unicode-literals-in-python-source-code

Reply:
Hi, @lukio:
yes, it's not necessary , thanks you for your feedback.